### PR TITLE
Added Gitter.

### PR DESCRIPTION
**Pull request by alejandronanez**
@jdjuan IDK about the right emojis for this section haha.

Let me know if you think I need to update something,

Thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,9 @@ Si quieres patrocinarnos puedes contactarnos [aquí](https://github.com/angular-
 
 Seguimos el código de conducta de la [comunidad de JavaScript](http://jsconf.com/codeofconduct.html), asegúrate de leerlo para disfrutar al máximo de Angular Medellín.
 
+## :speech_balloon: Únete a nuestro chat :speech_balloon:
+Estamos en Gitter! Puedes unirte [aquí](https://gitter.im/angular-medellin/Lobby?utm_source=share-link&utm_medium=link&utm_campaign=share-link)
```
Comment (jdjuan) [inline README.md]: I'd like to use a more challenging narrative here 😋  So what do you think of:

---
Todos saben que las comunidades más cool 😎 tienen un chat privado. ¡Nosotros también! Si quieres conocer las últimas noticias y estar al tanto de todas las novedades, **¡únete [aquí](https://gitter.im/angular-medellin/Lobby?utm_source=share-link&utm_medium=link&utm_campaign=share-link)!**

Comment (jdjuan) [inline README.md]: As to the emojis you can leave the ones you already have! 👍 but then replace the one used on the feedback title for 📢 

📢  ¿Qué opinas de la comunidad? 📢

**Comment by alejandronanez** [comment]
@jdjuan All done. Also good call about the Angular Commit conventions! Can you please update the message when you do the merge to master (I totally forgot to do it)? (You can modify the comment when you do `Squash Merge`)

Maybe we can add [Commitizen CLI](https://github.com/commitizen/cz-cli) + a hook to standarize git commit messages in the future? This should be a really good/not so hard for someone interested on contributing to OSS for the first time, what do you think? Maybe create a label like `First PR` - `Beginners Only` and ask for this change.

**Comment by jdjuan** [comment]
@alejandronanez I will do so when I squash. Thank you for the update 👍 🥇 

- I tried Commitizen at work, it worked, but users using a Git Client could not take advantage of it 😞. I guess it would be most of us. And on the other hand, there are people in our community that have barely ever made a PR in their lives, asking them to install Commitizen or a commit hook would be overkill for them.

- Yet we DO need a `CONTRIBUTING.md` in our community where we can just outline the steps of how to contribute and list the `Angular Commit Standards`, expecting them to follow it in most cases. We can talk to our community about it in the next meetup.

Hey @alejandronanez in general, thanks for your active support and participation, you've been a key member 🏅

**Comment by alejandronanez** [comment]
> Yet we DO need a CONTRIBUTING.md in our community where we can just outline the steps of how to contribute and list the Angular Commit Standards

Oh yeah we do need that!

> ...there are people in our community that have barely ever made a PR in their lives, asking them to install Commitizen or a commit hook would be overkill for them.

Yeah, you're right maybe it's quite complicated =[

**Pull request merged**